docs/navigate.rst

```diff
@@ -266,3 +274,9 @@ It displays the **default route domain** for the partition to which the Virtual
 If you encounter this issue, use the 'zoom out' option on your web browser. For example, on Windows, hold the **Ctrl** key, and then click **-** (the dash/minus key).
 
 ------
+
+**Q. VIP MAC masquerade Address might display old MAC on the visibility dashboard**
```

Comment:
**Q. VIP MAC masquerade addresses might display an old MAC address on the visibility dashboard**